src/_shared/js/capi-multiple.js

```diff
@@ -137,29 +134,32 @@ function editionLink (host, edition, adType) {
 }
 
 // Uses cAPI data to build the ad content.
-function buildFromCapi (host, cardsInfo, adType, modifyCardFn) {
+function buildFromCapi (host, cardsInfo, adType, generateLogo) {
```

Comment:
It should probably be more generic than that. Say you get a function that receives the card node and the current index as an argument:
```
function buildFromCapi (host, cardsInfo, adType, customise)`
function customise(card, index)
```